docs/architecture/cloud-native/combine-containers-serverless-approaches.md

```diff
@@ -31,17 +32,15 @@ Choose which worker runtime you want from the following options:
 - `node` (JavaScript)
```

Comment:
Per https://github.com/dotnet/docs/issues/17529, you may want to mention the presence of more runtime availability? IMO, instead of mentioning all the runtimes, it may be worth adding "and more" as text and reference it to the link mentioned in the issue.

Reply:
Added